eng/common/scripts/Service-Level-Readme-Automation.ps1

```diff
@@ -0,0 +1,255 @@
+<#
+.SYNOPSIS
+Update unified ToC file for publishing reference docs on docs.microsoft.com
+
+.DESCRIPTION
+Given a doc repo location and a location to output the ToC generate a Unified
```

Comment:
I don't think this description is correct for this script. This seems like it is from the ToC script.